src/components/clientDashboard/ProfileLayout.tsx

```diff
@@ -1,50 +1,33 @@
-import { Alert, AlertProps, AlertTitle, Box, Grid, Stack } from '@mui/material';
-import { isEmpty } from 'lodash-es';
+import { Box, Grid, Stack } from '@mui/material';
 
+import ClientAlerts from '@/modules/client/components/ClientAlerts';
 import ClientCustomDataElementsCard from '@/modules/client/components/ClientCustomDataElementsCard';
 import ClientEnrollmentCard from '@/modules/client/components/ClientEnrollmentCard';
 import ClientProfileCard from '@/modules/client/components/ClientProfileCard';
 import { ClientFieldsFragment } from '@/types/gqlTypes';
 
 export interface Props {
   client: ClientFieldsFragment;
-  notices?: ({
-    id: string;
-    content: React.ReactNode;
-    title?: React.ReactNode;
-  } & Omit<AlertProps, 'children'>)[];
 }
 
-const ProfileLayout: React.FC<Props> = ({ client, notices = [] }) => {
+const ProfileLayout: React.FC<Props> = ({ client }) => {
   const canViewEnrollments = client.access.canViewEnrollmentDetails;
+  const canViewClientAlerts = client.access.canViewClientAlerts;
   return (
     <Box data-testid='clientProfile'>
       <Grid container spacing={2}>
-        {!isEmpty(notices) && (
-          <Grid item xs={12}>
-            <Grid container spacing={2} justifyContent='space-between'>
-              {notices.map(({ content, title, id, ...props }) => (
-                <Grid item xs={12}>
-                  <Alert variant='filled' {...props} key={id}>
-                    {title && <AlertTitle>{title}</AlertTitle>}
-                    {content}
-                  </Alert>
-                </Grid>
-              ))}
-            </Grid>
-          </Grid>
-        )}
         <Grid item md={12} lg={canViewEnrollments ? 6 : 8}>
           <Stack gap={2}>
             <ClientProfileCard client={client} />
             <ClientCustomDataElementsCard client={client} />
           </Stack>
         </Grid>
-        {canViewEnrollments && (
-          <Grid item md={12} lg={6}>
-            <ClientEnrollmentCard client={client} />
-          </Grid>
-        )}
+        <Grid item md={12} lg={6}>
+          <Stack gap={2}>
+            {canViewClientAlerts && <ClientAlerts client={client} />}
```

Comment:
Is it ok that right now Alerts are attached to Clients and fetched in the same query? Or should they be more like Enrollments and fetched in a separate query?